posts/_markdown/calibration.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Calibrating Error Mitigation with Mitiq
+date: 3 April 2023
+author: Nate Stemen
+---
+
+# What is error mitigation calibration?
+
+Error mitigation offers potential improvements in the accuracy of measured expectation values, but with the myriad of mitigation strategies, and workflows to choose from, it can often be hard to know where to start.
```

Comment:
```suggestion
Error mitigation improves the performance of quantum programs, but with a myriad of mitigation strategies, and workflows to choose from, it can often be hard to know where to start.
```